bigtable_input.py

```diff
@@ -0,0 +1,91 @@
+"""Read Minigo game examples from a Bigtable.
+"""
+
+from google.cloud import bigtable
+from google.cloud.bigtable import row_filters
+import logging
+import os
+import struct
+import tensorflow as tf
+
+
+_project_name = os.environ['PROJECT']
+_instance_name = os.environ['CBT_INSTANCE']
+_table_name = os.environ['CBT_TABLE']
+
+_bt_table = bigtable.Client(
+    _project_name).instance(
+        _instance_name).table(
+            _table_name)
+
+_tf_table = tf.contrib.cloud.BigtableClient(
+    _project_name,
+    _instance_name).table(
+        _table_name)
+
+
+_features = {
+    'x': tf.FixedLenFeature((), tf.string, ''),
```

Comment:
Can you add "TODO(sethtroisi): Unify with preprocessing"

Reply:
Will do. In fact, now that I consider it, I'm not sure it's necessary to have `_features` in here, since at present, the Bigtable client simply passes back strings, and the parsing happens up in `preprocessing.py` anyway.